### PR TITLE
[Chore]: change bundler & add internal module

### PR DESCRIPTION
**Pull request by nonzzz**
## Check List

---

- [x] Remvoe cli part.
- [x] Add internal for each sub package.
- [x] Clean and light the code.  

## Change information

---

Currently, internal/plugins logic just a temporary solution. After `no-bump` upgrade. It also should refacotr.

### CODE REVIEW

**Comment by zaunist** [comment]
deploy failed?

**Comment by nonzzz** [comment]
> deploy failed?

my mistake. I change the command.

File: packages/website/package.json [hunk 1]
```diff
@@ -4,14 +4,13 @@
   "description": "A Website ",
   "main": "index.js",
   "scripts": {
-    "dev": "non dev",
-    "build": "non build"
+    "dev": "vite",
+    "prod": "vite build"
```
Comment (zaunist) [inline packages/website/package.json:8]: 这里把 build 命令改了，所以 netlify 的构建命令也要修改吧。

image

**Comment by nonzzz** [comment]
This pr is to get rid of meaningless modules. In this pr. I will use `no-bump` replace past bundle logic. You can view the past version Makefile. we should run `yarn` then build cli part and run `yarn clean` then re run `yarn`. I don't think it's necessary it adds complexity to the project.

File: packages/website/package.json (hunk 1 above)
Comment (nonzzz) [reply packages/website/package.json:8]: 我已经更改了netlify 的命令 @zaunist

**Comment by nonzzz** [comment]
I decide add internal module. Only do the common bundle or service logic. Compared to `packages/cli`, internal module is light enough

**Comment by zaunist** [comment]
> This pr is to get rid of meaningless modules. In this pr. I will use `no-bump` replace past bundle logic. You can view the past version Makefile. we should run `yarn` then build cli part and run `yarn clean` then re run `yarn`. I don't think it's necessary it adds complexity to the project.

What about after the PR? We no longer need to build the cli first, right?

**Comment by nonzzz** [comment]
> What about after the PR? We no longer need to build the cli first, right?

Yes, we no longer need cli. only use `internal` module and `no-bump` package replace it. you can view the diff. It's light.

**Comment by zaunist** [comment]
> > What about after the PR? We no longer need to build the cli first, right?
> 
> Yes, we no longer need cli. only use `internal` module and `no-bump` package replace it. you can view the diff. It's light.

I noticed that a lot of files were removed from the cli package, which is a great change.

**Comment by nonzzz** [comment]
Reasons for we don't need `cli`

- generator `volar` file only use for sub package named `vue`. But it be a base ability. 
- generator `full-module` entry. If you view cli part. you can find it's only work with package `vue`. After we refactor it. you can see `internal/helpers/gen.ts` can gen a universal `full-module` for each sub module.
- command `dev` and `build`. But I don't think sites should be placed in packages. With next pr i will refactor the website part.

So. that's why we don't need cli @zaunist

**Comment by nonzzz** [comment]
close #235

**Pull request merged**